app/src/pages/tracing/SpansTable.tsx

```diff
@@ -86,7 +86,7 @@ export function SpansTable(props: SpansTableProps) {
               span: node {
                 spanKind
                 name
-                statusCode
+                propagatedStatusCode
```

Comment:
I would recommend that we don't propagate on the spans table, since we also don't propagate the token counts. That way we would maintain a consistent methodology.
```suggestion
                statusCode
```

Reply:
Agreed, on the spans table you actually trying to find spans with bad status codes. It's one of the main filtering criteria that I would imagine you would have. Let's leave the spans alone and think of them in isolation here.